auth/service/service/service.go

```diff
@@ -56,6 +55,10 @@ type Service struct {
 	authClient         *Client
 	userEventsHandler  events.Runner
 	deviceCheck        apple.DeviceCheck
+	appValidator       *appvalidate.Validator
```

Comment:
I feel like these should be interfaces so they can be easily swapped out for tests.

Reply:
Rational behind these is the Validator itself contains only interfaces.

Reply:
Ah, fair enough then.